gnovm/stdlibs/std/context.go

```diff
@@ -17,4 +18,6 @@ type ExecContext struct {
 	OrigSend      std.Coins
 	OrigSendSpent *std.Coins // mutable
 	Banker        BankerInterface
+	EventLogger   *sdk.EventLogger
+	EmittedEvents abci.EventString
```

Comment:
@notJoon 

Why do you even need to introduce `EmittedEvents` to the `ExecContext`, if you have the `EventLogger`?

Can't you simply append the event to the `EventLogger` using `EventLogger.EmitEvent(...)` and later retrieve it using `EventLogger.Events()`?

Reply:
Yes this part is not really necessary. I added this field as a handler to save it as an `EventString` so I can use it separately. removed